### DISCUSSION TITLE
Streaming Multiple Synchronized Video Sources

### DISCUSSION THREAD

**Discussion by errollgarner** [PlusToolkit/PlusLib#774] [Q&A] [answered]
Is it possible to stream multiple video sources from a single PlusServer instance?

**Comment by jamesobutler** [comment, marked as answer]
Yes. I have multiple ultrasound devices specified in the XML config file used to start PlusServer. In that config file I also have multiple PlusOpenIGTLinkServer sections that each contain a video source that I stream over to 3D Slicer for viewing. Thereby viewing multiple active ultrasound streams at once in 3D Slicer.

**Reply by errollgarner** [reply]
Thank you for your help!
